react/features/prejoin/components/Prejoin.js

```diff
@@ -328,14 +309,24 @@ class Prejoin extends Component<Props, State> {
                 videoTrack = { videoTrack }>
                 {showJoinActions && (
                     <div className = 'prejoin-input-area-container'>
-                        <div className = 'prejoin-input-area'>
+                        {!walletSynced && timeout
```

Comment:
we no longer need the timeout, because on mobile browsers there is no wallet anyway and it will be never detected